src/unity/toolkits/evaluation/metrics.cpp

```diff
@@ -145,7 +214,7 @@ variant_map_type compute_classifier_metrics_from_probability_vectors(
   auto new_metrics_end = std::remove(metrics.begin(), metrics.end(),
                                      "confusion_matrix");
   if (new_metrics_end != metrics.end()) {
-    // Don't expose get_evaluator_metric() to "report_by_class".
+    // Don't expose get_evaluator_metric() to "confusion_matrix".
```

Comment:
This is just opportunistically cleaning up a copy-and-paste error in the comments